e2e/ansible/playbooks/hyperconverged/test-k8s-pvc-deletion/k8s_delete_pvc.yml

```diff
@@ -21,176 +24,205 @@
 
-       - name: 1a) Fetch the node count from stdout
-         set_fact:
-            node_count: " {{ node_out.stdout}}"
-
-# Attempting to create more replicas than the number of nodes
-       - name: 1c) Replace the replica count in storage class file
-         replace:
-           path: storage-class.yaml 
-           regexp: 'openebs.io/jiva-replica-count: "1"'
-           replace: 'openebs.io/jiva-replica-count: "{{ (node_count) |int+1}}"'
-           backup: yes
-
-       - name: 2) Get $HOME of K8s master for kubernetes user
-         shell: source ~/.profile; echo $HOME
-         args:
-           executable: /bin/bash
-         register: result_kube_home
-         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+        - name: 2a) Fetch the node count from stdout
+          set_fact:
+             node_count: " {{ node_out.stdout}}"
 
 #Copying openebs-test storage class yaml file to k8s master
-       - name: 2a) Copy the storage classes yaml to k8s master
-         copy:
-           src: "{{ sc_def }}"
-           dest: "{{ result_kube_home.stdout }}"
-         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
-
-       - name: 3) Check whether maya-apiserver pod is deployed
-         shell: source ~/.profile; kubectl get pods | grep maya-apiserver
-         args:
-           executable: /bin/bash
-         register: result
-         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
-         until: "'Running' in result.stdout"
-         delay: 120
-         retries: 5
+        - name: 3) Copy the storage classes yaml to k8s master
+          copy:
+            src: "{{ sc_def }}"
+            dest: "{{ result_kube_home.stdout }}"
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+# Attempting to create more replicas than the number of nodes
+        - name: 3a) Replace the replica count in storage class file
+          replace:
+            path: storage-class.yaml
+            regexp: 'openebs.io/jiva-replica-count: "3"'
+            replace: 'openebs.io/jiva-replica-count: "{{ (node_count) |int+1}}"'
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+        - name: 4) Check whether maya-apiserver pod is deployed
+          include_tasks: "{{ansible_env.HOME}}/{{utils_path}}/deploy_check.yml"
+          vars:
+             ns: openebs
+             app: openebs-api
+
+        - name: 5) Create test specific namespace
+          shell: source ~/.profile; kubectl create ns {{ namespace }}
+          args:
+            executable: /bin/bash
+          delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
 
 #create the storage class 'openebs-test'
-       - name: 4) Create the storage class in K8s master
-         shell: kubectl apply -f "{{ sc_def }}"
-         args:
-           executable: /bin/bash
-         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+        - name: 6) Create the storage class in K8s master
```

Comment:
@gprasath Can you include the task to deploy applications?

Reply:
Addressed.